ext/phar/makestub.php

```diff
@@ -48,8 +49,10 @@
   +----------------------------------------------------------------------+
 */
 
-static inline void phar_get_stub(const char *index_php, const char *web, size_t *len, char **stub, const int name_len, const int web_len)
+static inline zend_string *phar_get_stub(const char *index_php, const char *web, const int name_len, const int web_len)
```

Comment:
```suggestion
static inline zend_string *phar_get_stub(const char *index_php, const char *web, size_t name_len, size_t web_len)
```

It could make sense to also rearrange the parameter order in master alongside this.

Reply:
Just a note that this will also require format change in strpprintf.